include/library/spdm_transport_tcp_lib.h

```diff
@@ -0,0 +1,146 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2025 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#ifndef SPDM_TCP_TRANSPORT_LIB_H
+#define SPDM_TCP_TRANSPORT_LIB_H
+
+#include "library/spdm_common_lib.h"
+
+#define LIBSPDM_TCP_ALIGNMENT 4
```

Comment:
Why ALIGNMENT is 4 ?

Reply:
Yes, this probably is not needed, as TCP itself has no alignment requirements.

Reply:
i tried to follow other transports & the emulator code. I have removed the alignment in the latest patch set.